jsonrpc/eth_endpoint.go

```diff
@@ -433,12 +427,12 @@ func (e *Eth) GetStorageAt(
 
 // GasPrice returns the average gas price based on the last x blocks
 // taking into consideration operator defined price limit
-func (e *Eth) GasPrice() (string, error) {
+func (e *Eth) GasPrice() (interface{}, error) {
```

Comment:
why empty interface is better? what user cases it solves?

Reply:
That is how all the endpoints are defined. The `jsonrpc` library uses reflection to decide which methods export. The convention for all the methods is to return `(interface{}, error)`. This change just respects that convention.